quicktests/overlaying/tests/functional/formatter.js

```diff
@@ -12,7 +12,17 @@ function run(svg, data, Plottable) {
     };
 
 
-  var big_numbers = data[0].slice(0, 5);
+
+  var deep_copy = function(from, to){
```

Comment:
write this function into the same place as makeRandomData is written so that you don't have to rewrite this function every time.